HW_1/task_1.py

```diff
@@ -0,0 +1,14 @@
+# Решение первой задачи из первого домашнего задания
+"""
+Задача 2: Найдите сумму цифр трехзначного числа.
+*Пример:*
+
+123 -> 6 (1 + 2 + 3)
+100 -> 1 (1 + 0 + 0) 
+"""
+
+
+num = int(input("Input three-digit number: "))
+print(f'Sum of digits = {int(num / 100)+int(num / 10) % 10 + int(num % 10)}')
```

Comment:
Проще использовать операторы целочисленного отделения. А вот при получении остатка функция `int` вообще не нужна, так как дробный результат может получиться только если делимое будет дробным
```suggestion
print(f'Sum of digits = {num // 100 + num // 10 % 10 + num % 10}')
```